libraries/core_libs/network/include/network/tarcap/shared_states/syncing_state.hpp

```diff
@@ -61,13 +60,14 @@ class SyncingState {
   void set_peer(const dev::p2p::NodeID &peer_id);
 
  private:
-  std::shared_ptr<PeersState> peers_state_{nullptr};
-
+  std::atomic<bool> deep_pbft_syncing_{false};
   std::atomic<bool> pbft_syncing_{false};
   std::atomic<bool> dag_syncing_{false};
 
   ExpirationCache<dev::p2p::NodeID> malicious_peers_;
 
+  const uint16_t deep_syncing_threshold_;
```

Comment:
naming... https://google.github.io/styleguide/cppguide.html#Constant_Names

Reply:
does not apply here as it is class member first and this part ` whose value is fixed for the duration of the program` does not apply here as this variable is always different per each instance

Reply:
if so, we where wrong https://stackoverflow.com/questions/67655378/what-is-the-proper-case-of-constant-class-member-fields-according-to-the-google this should not apply

Reply:
based on single stackoverflow answer with single upvote ? :) I guess I can live with it, but if I forget about all the rules I think it is isier to read if I know straight away which class member is constant and which is not

Reply:
I am taking it back, you are right, it should be kDeepp......